pkg/license/multichannel.go

```diff
@@ -40,11 +43,20 @@ func VerifyAndUpdateLicense(log *logger.CLILogger, license *kotsv1beta1.License,
 		return nil, nil
 	}
 	if isAirgap {
+		log.ActionWithSpinner("Verifying if channel slug %q is allowed by the license", preferredChannelSlug)
```

Comment:
i don't think this message is necessary. we can just print an error message if it's not allowed.